infrastructure/helm-chart/charts/controller/templates/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airy-controller
```

Comment:
I'm always up for dropping the prefix _but_ in this case I agree it's needed. I'd specify even more, something like `airy-core-controller`